lib/chat/bloc/provider/ChatProvider.dart

```diff
@@ -34,10 +32,10 @@ class _Constants {
 }
 
 class _Strings {
```

Comment:
Rename it to _LocalisedStrings 
```suggestion
class _LocalisedStrings {
```

Reply:
👍

Reply:
Renamed